Containers/mastercontainer/Caddyfile

```diff
@@ -14,18 +14,17 @@
     servers {
         protocols h1 h2 h2c
     }
+
+    on_demand_tls {
+        ask http://localhost:9876/
+    }
```

Comment:
why as a global option?

Reply:
beccause this is a global option, I think?

Reply:
Based on https://caddy.community/t/serving-tens-of-thousands-of-domains-over-https-with-caddy/11179 this seems correct